src/OldManBot/OldMan.ts

```diff
@@ -17,3 +18,5 @@ OldMan.config({
 
 process.on('exit', () => OldMan.stop())
 process.on('SIGINT', () => OldMan.stop())
+process.on('uncaughtException', error => winston.error(error))
```

Comment:
Helpful reminder here. Uncaught whatevers need to check if logs are flushed before exiting. Otherwise process will close before the log is executed and transported.

Reply:
https://github.com/winstonjs/winston#awaiting-logs-to-be-written-in-winston

Reply:
I know Oldman is structured to be a process running inside of a process, essentially so this might not apply.

Reply:
Good catch, I'll push a new commit.